packages/compass-workspaces/src/index.spec.tsx

```diff
@@ -132,16 +132,18 @@ describe('WorkspacesPlugin', function () {
 
     userEvent.click(screen.getByRole('tab', { name: 'coll1' }));
 
-    await waitFor(() => {
-      expect(screen.getByRole('tab', { name: 'coll3' })).to.have.attribute(
-        'aria-selected',
-        'false'
-      );
-      expect(screen.getByRole('tab', { name: 'coll1' })).to.have.attribute(
-        'aria-selected',
-        'true'
-      );
-    });
+    // TODO: The connection is disconnected somehow/somewhere which triggers cleanup and
+    // tabs are removed and these assertions fail. Need to figure this out!
```

Comment:
Is this still outstanding?

Reply:
Weirdly enough, doesn't look like it's failing in test-electron, so somehow this is runtime specific? It's very unexpected tbh, so we should really try to get to the bottom of this, ping me if you want to poke around in this together